dbServer/Dockerfile

```diff
@@ -29,6 +32,6 @@ RUN rm -rf ${GRAPHDB_HOME}/data
 COPY ./data ${GRAPHDB_HOME}/data
 COPY ./conf ${GRAPHDB_HOME}/conf
 
-CMD ["-Dgraphdb.home=/graphdb/home","-Dgraphdb.logback=conf/logback.xml","-Dgraphdb.workbench.external-url=http://localhost/modelujeme/sluzby/dbServer","-Dgraphdb.external-url=http://localhost/modelujeme/sluzby/dbServer"]
+CMD ["-Dgraphdb.home=/graphdb/home","-Dgraphdb.logback=conf/logback.xml","-Dgraphdb.workbench.external-url=$GRAPHDB_WORKBENCH_EXTERNAL_URL","-Dgraphdb.external-url=$GRAPHDB_EXTERNAL_URL"]
```

Comment:
Passing the ENV to the CMD should work - in that case it should be (hopefully) replaced by the one from docker compose.